cli/cmd/completion.go

```diff
@@ -29,15 +29,22 @@ func newCmdCompletion() *cobra.Command {
   source <(linkerd completion zsh)
 
   # zsh on osx / oh-my-zsh
-  linkerd completion zsh > "${fpath[1]}/_linkerd"`
+  linkerd completion zsh > "${fpath[1]}/_linkerd"
+
+  # Fish:
```

Comment:
Small suggestion, but just keeping it consistent with the above help text

```suggestion
  # fish
```